adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala

```diff
@@ -396,17 +415,34 @@ case class RDDBoundFeatureRDD private[rdd] (
     sqlContext.createDataset(rdd.map(FeatureProduct.fromAvro))
   }
 
+  /**
+   * Replaces the sequence dictionary attached to a RDDBoundFeatureRDD.
+   *
+   * @param newSequences The new sequence dictionary to attach.
+   * @return Returns a new RDDBoundFeatureRDD with the sequences replaced.
+   */
   def replaceSequences(newSequences: SequenceDictionary): FeatureRDD = {
```

Comment:
This method is defined with scaladoc in `GenomicDataset`, could we use `override def` instead?